### ISSUE TITLE
Skip errors

### ISSUE THREAD

**Issue by CWBudde** [techouse/sqlite3-to-mysql#86]
In case of SQL errors, I would like to skip this table and move on to the next table. This might end in a corrupt database, but it's sometimes easier to fix the errors manually. In my case it's just an issue with an index, which I can easily recreate.

In order to make it clear, I would prefer a custom command-line option "ignore-error" or "force-conversion" with the description that errors will be ignored and the conversion might be incomplete in the end.

I'm willing to implement this by myself in a fork. If so, I'd open a pull request.

**Comment by techouse** [comment]
Hey!

Have you tried any of these options:
```
  -X, --without-foreign-keys      Do not transfer foreign keys.
  -W, --ignore-duplicate-keys     Ignore duplicate keys. The default behavior
                                  is to create new ones with a numerical
                                  suffix, e.g. 'exising_key' ->
                                  'existing_key_1'
```

Othrwise I'm not keen on implementing a data corruption feature.

If you want, you can submit a PR.

**Comment by CWBudde** [comment]
Yes, I have tried both. No success. The PR is on its way. I'm currently testing it.

**Comment by techouse** [comment]
> I'm currently testing it.

Don't forget about unit tests 😇
